labop/many_to_one_sample_map.py

```diff
@@ -0,0 +1,38 @@
+"""
+The ManyToOneSampleMap class defines the functions corresponding to the dynamically generated labop class ManyToOneSampleMap
+"""
+
+import xarray as xr
+
+import labop.inner as inner
+from labop.sample_map import SampleMap
+
+
+class ManyToOneSampleMap(inner.ManyToOneSampleMap, SampleMap):
```

Comment:
What does ManyToOneSampleMap represent?  (Is this class a recent addition to the data model or have I just been completely unaware of its existence?)

Reply:
I'm going to remove these.  I think its cruft that didn't make it into use.